blog/_posts/2025-08-05-aks-mcp-server.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Announcing the AKS-MCP Server: Unlock Intelligent Kubernetes Operations"
+description: "Learn about AKS-MCP server, the latest open-source tooling to Unlock intelligent automation 
+by standardizing and streamlining the way AI agents interact with your AKS cluster."
+date: 2025-08-05
+authors: Pavneet Ahluwalia
+categories: 
+  - developer
+  - ai
+  - operations
+tags:
+  - ai
+  - mcp
+  - open-source
+---
+
+We're excited to announce the launch of the **AKS-MCP Server**.
+An open source Model Context Protocol (MCP) server designed to make your Azure Kubernetes Service (AKS)
+clusters AI-native and more accessible than ever for developers, SREs, and platform engineers.
+
+AKS-MCP isn't just another integration layer.
+It empowers cutting-edge AI assistants (such as **Claude**, **Cursor**, and **GitHub Copilot**)
+to interact with AKS through a secure, standards-based protocol—opening new possibilities for
+automation, observability, and collaborative cloud operations.
+
+We believe users should have the flexibility to leverage the AI tooling of their choice with AKS. For instance if you have a strong
+affinity for Claude code or Github-Copilot or some other agent for your workflows,
+you can plug-in AKS-MCP Server and unlock intelligent interactions, automation and troubleshooting for your AKS environments.
+
+![aks-mcp-github](/assets/images/aks-mcp/aks-mcp-github.png)
+
+## The Problem: Why Do We Need MCP Now?
+
+The biggest pain point facing modern AI assistants is not their reasoning or language abilities,
+but the fragmented, brittle context in which they operate. Organizations today are struggling with:
+
+- **Siloed information and workflows**: AI can only act on what it can see—often, just a fraction of what's truly relevant.
+- **Complex, expensive integrations**: Every new data source, tool, or AI agent means more custom connectors, patches, and technical debt.
+- **Stunted automation and insight**: When assistants lack rich, real-time access to infrastructure, their utility is massively limited.
+
+## The Solution: AKS-MCP's Open, Secure Protocol
+
+AKS-MCP answers this challenge as a universal, protocol-first bridge between AI agents and AKS. Built on the open Model Context Protocol (MCP), it delivers:
+
+- **One protocol, limitless context**: Replace dozens of brittle integrations with a secure, standard interface—making all your Kubernetes data, events, and actions available to any AI assistant.
```

Comment:
Would be good to dive deeper at least to a L100 design on how this feature works and what can I observe? for example, questions I have a yesteryear user of AKS and reader of blog - is it still based on making API calls to ARM or Kubernetes API? How should I think of the scale of requests for simple vs complex queries? How is authn and token management handled specifically?